command/agent/consul/catalog_testing.go

```diff
@@ -121,7 +136,34 @@ func (c *MockAgent) Self() (map[string]map[string]interface{}, error) {
 	defer c.mu.Unlock()
 	c.hits++
 
-	s := map[string]map[string]interface{}{
+	version := "1.9.5"
+	build := "1.9.5:22ce6c6a"
+	if c.ent {
+		version = "1.9.5+ent"
+		build = "1.9.5+ent:22ce6c6a"
+	}
+
+	stats := make(map[string]interface{})
+	if c.ent {
+		if c.namespaces {
+			stats = map[string]interface{}{
+				"license": map[string]interface{}{
+					"features": "Namespaces,",
+				},
+			}
+		}
+	}
+
+	return map[string]map[string]interface{}{
+		"Config": {
+			"Datacenter": "dc1",
+			"NodeName":   "x52",
+			"NodeID":     "9e7bf42e-a0b4-61b7-24f9-66dead411f0f",
+			"Revision":   "22ce6c6ad",
+			"Server":     true,
+			"Version":    version,
+		},
+		"Stats": stats,
```

Comment:
This field isn't documented in the [`/v1/agent/self`](https://www.consul.io/api-docs/agent#read-configuration) docs. Do we know if that's just a docs miss or is this an unstable API field?

Reply:
It's [hard-coded](https://github.com/hashicorp/consul/blob/v1.9.5/agent/agent_endpoint.go#L95) as part of the response struct - I think it's just outdated docs at this point.